src/main/diagnostics/__tests__/diagnostics.spec.ts

```diff
@@ -0,0 +1,67 @@
+import yaml from 'yaml';
+
+import { Diagnostics } from '../diagnostics';
+
+describe(Diagnostics, () => {
+  const diagnosticsTable = `
+diagnostics:
+  last_update: 2022-08-15T16:00+00:00
+  categories:
+    - title: Utilities
+      checks:
+        - id: RD_BIN_IN_BASH_PATH
+          documentation: path#rd_bin_bash
+          description: "The ~/.rd/bin directory has not been added to the PATH, so command-line utilities are not configured in your bash shell."
+          mute: false
+          fixes:
+            description: "You have selected manual PATH configuration, you can let Rancher Desktop automatically configure it."
+        - id: RD_BIN_SYMLINKS
+          documentation: path#rd_bin_symlinks
+          description: "Are the files under ~/.docker/cli-plugins symlinks to ~/.rd/bin?"
+          mute: false
+          fixes:
+            description: "Replace existing files in ~/.rd/bin with symlinks to the application's internal utility directory"
+    - title: Networking
+      checks:
+        - id: CONNECTED_TO_INTERNET
+          documentation: path#connected_to_internet
+          description: "The application cannot reach the general internet for updated kubernetes versions and other components, but can still operate."
+          mute: false
+`;
+  const diagnostics = new Diagnostics(yaml.parse(diagnosticsTable).diagnostics);
+
+  test('it finds the categories', () => {
+    expect(diagnostics.getCategoryNames()).toMatchObject(['Utilities', 'Networking']);
```

Comment:
Using `toMatchObject` here seems odd (it checks that each item matches the given one, _in order_); did you intend to use [`expect.arrayContaining`](https://jestjs.io/docs/expect#expectarraycontainingarray)? Or pull in `jest-extended` and use [`toIncludeSameMemebers`](https://www.npmjs.com/package/jest-extended#toincludesamemembersmembers)?

This applies to other uses of `toMatchObject` as well.

Reply:
Done